src/components/HelpIconModal.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <div class="help-icon-modal">
+    {{ title }}
+    <p-icon class="help-icon-modal__icon" v-bind="attrs" :icon="icon" solid @click.stop.prevent="open" />
+
+    <p-modal v-model:show-modal="showModal" :title="title">
+      <slot name="description" />
+
+      <slot name="content" />
```

Comment:
These can probably both be the default slot

```suggestion
      <slot />
```